bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBS/ZZ2l2nujj/ZZ2l2nujj_ewk_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model sm-ckm_no_b_mass
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = p
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate  p p > z z j j QED=4 QCD=0, z > l+ l-, z > vl vl~
```

Comment:
I would model this differently.
p p > z vl vl~ j j QED=5 QCD=0, z > l+ l-.
neutrino momentum (MET) doesn't necessarily have to be bound to Z mass poles so this way you would at least gain something more sensitivity but this is fine. Just dumping some ideas that one might consider in the future.